### PR TITLE
Estimator eligibility questions update

### PR DESCRIPTION
**Pull request by SelvaRC**

### CODE REVIEW

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC1/activity_eligibility/HVAC1_activity_eligibility_parameters.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ class HVAC1_engaged_ACP(Variable):
     default_value = True
     definition_period = ETERNITY
     metadata = {
-        'display_question' : 'Was or will an Accredited Certificate Provider be engaged before the implementation date?',
+        'display_question' : 'will an Accredited Certificate Provider be engaged before the implementation date?',
```
Comment (k8mckay) [inline openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC1/activity_eligibility/HVAC1_activity_eligibility_parameters.py:67]: This one needs a capital first letter 'Will an .. '

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC2/activity_eligibility/HVAC2_activity_eligibility_parameters.py [hunk 2]
```diff
@@ -79,7 +79,7 @@ class HVAC2_residential_building(Variable):
     default_value = False
     definition_period = ETERNITY
     metadata = {
-        'display_question' : 'Has the new End-User equipment been installed in a residential building?',
+        'display_question' : 'Will the new End-User equipment been installed in a residential building?',
```
Comment (k8mckay) [inline openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC2/activity_eligibility/HVAC2_activity_eligibility_parameters.py:82]: This ones needs to be   '... equipment be installed ... ' (not been)

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/SYS2/activity_eligibility/SYS2_activity_eligibility_parameters.py [hunk 3]
```diff
@@ -60,7 +60,7 @@ class SYS2_legal_disposal(Variable):
     default_value = True
     definition_period = ETERNITY
     metadata = {
-      'display_question' : 'Has the decommissioned pool pump been removed in accordance with relevant safety standards and legislation?',
+      'display_question' : 'Will the decommissioned pool pump been removed in accordance with relevant safety standards and legislation?',
```
Comment (k8mckay) [inline openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/SYS2/activity_eligibility/SYS2_activity_eligibility_parameters.py:63]: "... pool pump be removed ..." (not been)

**Review by k8mckay — COMMENTED** [review]
There are a couple of typos to update if you could make those changes and let me know in here.

**Comment by SelvaRC** [comment]
@k8mckay i have updated the requested changes and tested :)

**Review by k8mckay — APPROVED** [review]

**Review by k8mckay — APPROVED** [review]

**Review by k8mckay — APPROVED** [review]

**Review by k8mckay — APPROVED** [review]

**Pull request merged**